youths/tests/test_gdpr_api.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from django.urls import reverse
+from rest_framework import serializers
+
+from youths.models import YouthProfile
+
+
+def test_disabled_gdpr_api_responds_with_404(settings, api_client, youth_profile):
+    settings.GDPR_API_ENABLED = False
+    response = api_client.get(
+        reverse("youths:gdpr", kwargs={"pk": youth_profile.profile.id})
+    )
+    assert response.status_code == 404
+
+    response = api_client.delete(
+        reverse("youths:gdpr", kwargs={"pk": youth_profile.profile.id})
+    )
+    assert response.status_code == 404
+
+
+def test_get_profile_information_from_gdpr_api(api_client, youth_profile, snapshot):
+    response = api_client.get(
+        reverse("youths:gdpr", kwargs={"pk": youth_profile.profile.id})
+    )
+
+    assert response.status_code == 200
+    snapshot.assert_match(response.json())
+
+
+@pytest.mark.parametrize("true_value", serializers.BooleanField.TRUE_VALUES)
```

Comment:
Just as a matter of principle, I would not use values from DRF but instead values explicitly defined for this test, because now if DRF's TRUE_VALUES change in some version this API will change too, but the test won't catch it. Yes, that will probably never happen in practice :D

Reply:
I shall consider using explicit values here.